### PR TITLE
 A Tiny JS World

### PR DESCRIPTION
**Pull request by Yanyshpolska**
### A Tiny JS World

[Demo](https://yanyshpolska.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/Yanyshpolska/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Yanyshpolska** [comment]
Self-check completed.

File: submissions/yanyshpolska/tiny-world/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  saying: "Hi there!",
+};
+
+const inhabitants = [dog, cat, woman, womanCat, man];
+const keys = ["species", "name", "gender", "legs", "hands", "saying"];
+
+inhabitants.map((inhabitant) => {
+  const arr = [];
+  keys.forEach((key) => arr.push(inhabitant[key]));
+  print(arr.join("; "));
```
Comment (OleksiyRudenko) [inline submissions/yanyshpolska/tiny-world/index.js:58]: Please use `Array#map` and `Array` methods chaining here. You will find `arr` variable redundant.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@Yanyshpolska well done!
One improvement and we are done.

**Review by OleksiyRudenko — APPROVED** [review]
@Yanyshpolska great job!

**Pull request merged**